ignite/handlers/checkpoint.py

```diff
@@ -52,7 +52,6 @@ class Checkpoint:
             Input of the function is `(engine, event_name)`. Output of function should be an integer.
             Default is None, global_step based on attached engine. If provided, uses function output as global_step.
             To setup global step from another engine, please use :meth:`~ignite.handlers.global_step_from_engine`.
-        archived (bool, optional): Deprecated argument as models saved by `torch.save` are already compressed.
```

Comment:
Same here. Keep `archived` and please add a note that it will be removed since v0.5.0